shared/test/dataDriver_test.js

```diff
@@ -506,23 +506,24 @@ describe("loop.DataDriver", () => {
       driver._processRecord("participant!theUserId", {
         timestamp: 1234567890123,
         value: {
-          name: "Cool Name"
+          participantName: "Cool Name"
         }
       });
 
       sinon.assert.called(dispatcher.dispatch);
       sinon.assert.calledWithExactly(dispatcher.dispatch,
         new actions.UpdatedParticipant({
-          name: "Cool Name",
+          participantName: "Cool Name",
           userId: "theUserId"
         }));
     });
   });
 
   describe("#_processRecord.presence", () => {
     it("should dispatch `UpdatedPresence` for presence record", () => {
+      clock.tick(1234567890123);
       driver._processRecord("presence!theUserId", {
-        timestamp: 1234567890123,
+        timestamp: 1234567889123,
```

Comment:
This should be a computed number as it's easy to look over the difference between this and the tick amount above.